### PR TITLE
Feature: Enable dev mode

### PR DESCRIPTION
**Pull request by SharonStrats**
This enables you to work standalone with the new auth.

To test: 
1.  Bring down branch 
2.  Change links to webid and address book to your own in index.ts
3. Npm run start

You should then see your address book at localhost:8080

Note: This is now in feature styles branch.  I will close it off if that is accepted otherwise I will come back to this and change the few things that need to change after auth refactor.

### CODE REVIEW

**Comment by bourgeoa** [comment]
- I had to add : `npm install @babel/core --save` to be able to `npm run start`
- below is the result : 
image

**Comment by SharonStrats** [comment]
> 

Hi Alain, You just need to change the Address book to your own inside index.ts.  This one is my own and it's a private resource.  Sorry I should have made a public one to show. I can do that if. you want, just ping me on Gitter and I'll do it later.

**Comment by bourgeoa** [comment]
May be like in solid/profile-pane Dev where you have a WebID form, you could add a contactsUrl one.

File: dev/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <title>profile-pane dev server</title>
```
Comment (theRealImy) [inline dev/index.html:5]:
```suggestion
    <title>contacts-pane dev server</title>
```

File: dev/index.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  await authn.authSession.handleIncomingRedirect();
+  const session = authn.authSession;
+  if (session.info.isLoggedIn) {
+    // Update the page with the status.
+    webId.innerHTML = "Logged in as: " + authn.currentUser().uri;
+  } else {
+    webId.innerHTML = "";
+  }
+}
+
```
Comment (theRealImy) [inline dev/index.ts:21]: I would clean up the space here.

**Comment by theRealImy** [comment]
I did not need the npm install @babel/core --save command but I did have to do npm install beforehand, which I think is normal the first time. 
Everything else works fine for me.

**Comment by timbl** [comment]
I find npm install works but with npm start I get a 3 Modue not Found errors, one like `ERROR in ./individual.js 5:0-38
Module not found: Error: Can't resolve './lib/forms' in '/Users/timbl/src/github.com/solid/solidos/workspaces/contacts-pane'
`

**Comment by theRealImy** [comment]
> I find npm install works but with npm start I get a 3 Modue not Found errors, one like `ERROR in ./individual.js 5:0-38 Module not found: Error: Can't resolve './lib/forms' in '/Users/timbl/src/github.com/solid/solidos/workspaces/contacts-pane' `

@timbl I think I solved that with simply npm run build...

**Review by angelo-v — APPROVED** [review]

**Pull request closed without merging**